{{cookiecutter.module_name|slugify}}/Makefile

```
@@ -0,0 +1,51 @@
+pyupgrade = pyupgrade --py39-plus $$(find inmanta_plugins tests -type f -name '*.py')
+flake8 = flake8 inmanta_plugins tests 
+black = black inmanta_plugins tests
+black_preview = black --preview inmanta_plugins tests
+isort = isort inmanta_plugins tests
+
+
+.PHONY: install
+install:
+	pip install -U --upgrade-strategy=eager pip setuptools wheel
+	pip install -U --upgrade-strategy=eager -e . -c requirements.txt -r requirements.dev.txt
+
+
+.PHONY: format
+format:
+	$(isort)
+	$(black_preview)
+	$(black)
+	$(flake8)
+	$(pyupgrade)
+
+
+.PHONY: pep8
+pep8:
+	$(flake8)
+	$(pyupgrade)
+
+SET_UP_MYPY_PLUGINS=mkdir -p mypy/out
+RUN_MYPY_PLUGINS=python -m mypy --namespace-packages -p {{cookiecutter.package_name}}
```

```suggestion
RUN_MYPY_PLUGINS=python -m mypy --namespace-packages -p inmanta_plugins.{{cookiecutter.inmanta_module_name}}
```
Did you test it? I would actually expect this